client/app/scripts/superdesk-archive/views/item-crops.html

```diff
@@ -0,0 +1,4 @@
+<div ng-repeat="crop in crop_sizes | orderBy:'name' track by crop.name" ng-if="item.renditions[crop.name]">
+    <span translate>{{crop.name}}<span>
+    <div class="rendition" sd-item-rendition data-item="item" data-ratio="ratio" data-rendition="{{:: crop.name}}"></div>
```

Comment:
Can `crop.name` change or not? In this line it is one-time bound, but in the line above it is not - which of the two should be updated?